inginious/frontend/task_dispensers/__init__.py

```diff
@@ -11,6 +12,16 @@ def __init__(self, task_list_func, dispenser_data, database, course_id):
         """
         pass
 
+    @abstractmethod
+    def get_weight(self, taskid):
```

Comment:
I'm not really quite sure this makes sense here, as there may be task dispensers that do not use tasks weights, as yours (adaptative testing). ``get_weight`` should not be part of the interface, and the concept should be encapsulated in the task dispenser.